client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -116,7 +116,17 @@ def process(self, instance):
                 instance,
                 self.job_info
             )
-        self.plugin_info = self.get_plugin_info()
+
+        # If the render file export happens locally,
+        # get the plugin info directly as a render job type.
+        creator_attribute = instance.data.get("creator_attributes")
+        if (
+            creator_attribute.get("render_target")
+            == "local_export_farm_render"
+        ):
+            self.plugin_info = self.get_plugin_info(job_type="render")
+        else:
+            self.plugin_info = self.get_plugin_info()
```

Comment:
This is very specific I feel to this particular case and I have my doubts about whether we want this at the level of the Abstract plug-in. _Especially_, because incorrectly `get_plugin_info` takes an extra argument on the Houdini plug-in.

We _should_ work towards a setup where we _can_ live without that extra argument so that it behaves like the base plug-in in argument signature. I'll take a quick look.

Reply:
Ah it does take `**kwargs` in it's definition. Not saying it's good, but it does work. I don't see a quick way to restructure this into something way more readable without initiating a larger redesign.

We'll need to re-think how we design and populate these stacked dependency graphs in a simple standardized manner @philippe-ynput because this code is starting to get very confusing.

I think we should work towards a way where can build a named stack of jobs, like `export`, `render`, `publish` and from the different addons can just target those individual jobs by name instead of guessing what is what. OpenJD will probably be at the core of that.